packages/attestation-service/migrations/20200506234935-attestation-v2.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+
+    try {
+      await queryInterface.removeColumn('Attestations', 'phoneNumber')
+
+      await queryInterface.addColumn('Attestations', 'identifier', {
+        type: Sequelize.STRING,
+      })
+
+      await queryInterface.removeIndex(
+        'Attestations',
+        ['account', 'phoneNumber', 'issuer'],
+        { fields: ['account', 'phoneNumber', 'issuer'], unique: true },
+        { transaction }
+      )
+
+      await queryInterface.addIndex(
+        'Attestations',
+        ['account', 'identifier', 'issuer'],
+        { fields: ['account', 'identifier', 'issuer'], unique: true },
+        { transaction }
+      )
+
+      await queryInterface.addIndex(
+        'Attestations',
+        ['createdAt'],
+        { fields: ['createdAt'], unique: true },
+        { transaction }
+      )
+
+      await transaction.commit()
+    } catch (error) {
+      await transaction.rollback()
+      throw error
+    }
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('Attestations')
```

Comment:
In the `down` method, one would generally specify the inverse of `up` in case one would like to reverse the migration. So i.e. you would probably want to 
- remove the `identifier` column
- add the `phoneNumber` column
- change the appropriate indicies

Reply:
gotcha -- that makes sense